jetty-server/src/main/config/etc/jetty-requestlog.xml

```diff
@@ -12,7 +12,7 @@
   <!-- =========================================================== -->
   <Set name="RequestLog">
     <New id="RequestLog" class="org.eclipse.jetty.server.AsyncNCSARequestLog">
-      <Set name="filename"><Property name="jetty.base" default="." /><Property>
+      <Set name="filename"><Property name="jetty.base" default="." />/<Property>
           <Name>jetty.requestlog.filePath</Name>
           <Deprecated>requestlog.filename</Deprecated>
           <Default><Property name="jetty.requestlog.dir" default="/logs"/>/yyyy_mm_dd.request.log</Default>
```

Comment:
We added the same slash between jetty.base and the property to jetty-deploy.xml for the same reason, and noone has complaining about that one.

https://github.com/eclipse/jetty.project/blob/jetty-9.3.9.v20160517/jetty-deploy/src/main/config/etc/jetty-deploy.xml#L42